checkout_session.go

```diff
@@ -203,7 +203,7 @@ type CheckoutSession struct {
 	Deleted                   bool                                      `json:"deleted"`
 	DisplayItems              []*CheckoutSessionDisplayItem             `json:"display_items"`
 	ID                        string                                    `json:"id"`
-	LineItems                 []*LineItem                               `json:"line_items"`
+	LineItems                 *LineItemList                             `json:"line_items"`
```

Comment:
Is that a thing? Not seeing `LineItemList` defined...

Reply:
nevermind i didn't have latest master...

Reply:
It's defined in `lineitem.go` https://github.com/stripe/stripe-go/blob/master/lineitem.go#L27-L32

All resources have a `<Resource>List` version. And so if you look at `Customer` it uses `SubscriptionList` for example: https://github.com/stripe/stripe-go/blob/master/customer.go#L117